### DISCUSSION TITLE
smooth analysis for ica-aroma

### DISCUSSION THREAD

**Discussion by JiaweiSun0512** [maartenmennes/ICA-AROMA#66] [Q&A] [answered]
Hi all, I have a question, in the preprocessing of ica-aroma, the data shoul be smoothed. But I subsequently want to do regional homogeneity (reho) and degree centrality (dc) analyses, and this step requires that the data is no smoothing. Do you have any ways to solve this?

**Comment by maartenmennes** [comment, marked as answer]
The smoothing is required for the ICA decomposition, so one way to do this is to run feat preprocessing as normal (including smoothing) and select the 'MELODIC data exploration'. This will run ICA. Then run the preprocessing again (from your original raw data!), now don't select the 'MELODIC data exploration' option, and you can set the smoothing to 0 for your purposes. When it's finished, copy the filtered_func.ica folder of the first run to the feat folder of the new preprocessing run. Then run AROMA as per the manual... The component selection will use the ICA decomposition based on the smoothed data, but the denoising will happen on your non-smoothed data.
